### ISSUE TITLE
Support `BigNumber` as `wordSize` in function `format`

### ISSUE THREAD

**Issue by bornova** [josdejong/mathjs#3142]
**Describe the bug**
Currently, the function `format` does not support a `BigNumber` as `wordSize`

**To Reproduce**
See following example:
```javascript
// With math.config({ number: 'BigNumber' })
math.evaluate(format(-830, { notation: "hex", wordSize: 32 }))  // Error: Option "wordSize" must be a number
```
In reference to: https://github.com/josdejong/mathjs/issues/3137#issuecomment-1909638351

**Comment by josdejong** [comment]
Thanks for reporting, fixed via c60f637 (not yet published)

**Comment by josdejong** [comment]
Fixed now in `v12.3.1`
